src/server/src/main/java/io/cassandrareaper/jmx/ClusterFacade.java

```diff
@@ -698,8 +698,7 @@ public boolean nodeIsAccessibleThroughJmx(String nodeDc, String node) {
    */
   public List<Snapshot> listSnapshots(Node host) throws ReaperException {
     try {
-      if (context.config.getDatacenterAvailability().isInCollocatedMode()
-          && context.jmxConnectionFactory.getHostConnectionCounters().getSuccessfulConnections(
+      if (context.jmxConnectionFactory.getHostConnectionCounters().getSuccessfulConnections(
```

Comment:
```suggestion
      if (!context.config.getDatacenterAvailability().isInCollocatedMode()
          && context.jmxConnectionFactory.getHostConnectionCounters().getSuccessfulConnections(
```

The check was indeed wrong as we need to make sure we're not in collocated (distributed) mode for the snapshots to be listed.
Listing snapshots involves a direct call to each node in the cluster which is most probably impossible in collocated mode.

Reply:
Hi @adejanovski 

seems fair. I've applied the change you suggested.

Thanks!